### ISSUE TITLE
Plot how learning rate varies along the epochs

### ISSUE THREAD

**Issue by dariogonle** [ultralytics/yolov5#2892]
## ❔Question
Hi,

 don't you think it would be nice to plot how learning rates varies along the epochs? Something like what it is shown below. I'am trying to train a model with adaptative learning rate, and it would help me to ensure  that the learning rate is being decreasing as I want.

If you don't have time I can code it, but I would need to know where is the best for doing it. Would it be easy to and the learning rate value of each epoch to results.txt? Where should I modify results.txt?

Thank you.

## Additional context
![image](https://user-images.githubusercontent.com/80393891/115701571-38373300-a368-11eb-9ac5-8193e3069e72.png)

**Comment by AndreaBrg** [comment]
Did you check the tensorboard plots? It's right there.

**Comment by glenn-jocher** [comment]
@dariogonle good news 😃! This feature is already implemented by default. Learning rates are displayed vs epochs in TensorBoard and W&B:

<img width="1055" alt="Screen Shot 2021-04-20 at 5 06 44 PM" src="https://user-images.githubusercontent.com/26833433/115707304-e3e38180-a36e-11eb-826f-80ff8ed119aa.png">

**Comment by dariogonle** [comment]
Sorry, I always look at local plots not to tensorboard. Thank you @AndreaBrg and @glenn-jocher.
